src/DynamoCoreWpf/UI/Prompts/EditWindow.xaml.cs

```diff
@@ -42,14 +42,19 @@ public partial class EditWindow
         }
         private void OnEditWindowPreviewKeyDown(object sender, KeyEventArgs e)
         {
-            if(e.Key == Key.Return ||e.Key == Key.Enter)
+            var expr = editText.GetBindingExpression(TextBox.TextProperty);
```

Comment:
Can we have a property called `CommitChangesOnReturn` and initialize it properly from the clients side.